### DISCUSSION TITLE
Font preview is too small for niche configurations and unorthodox modus operandi

### DISCUSSION THREAD

**Discussion by KodeXall** [jonaskohl/CapsLockIndicator#132] [Ideas]
The box showcasing the text of Notifications is too small. Because if as user for whatever reason decides to utilize font set too Size e.g. 72, then only fragments of few letters are visible at the

**Notification > Font**

To avert this issue, at least to some extent, that area should be enlarged a little vertically and greatly  in the horizontal dimension. I propose moving it to the bottom, i.e. below **_Overlay position_** and _**Border thickness**_ options, thus making it twice as wide as it is now

And why would anyone use such absurd big font? Well, if you are using a 2160p widescreen monitor and a zoom-in feature like Microsoft Screen Magnifier afar from monitor, then a bloated Notification is a must in order for CapsLock Indicator to retain its usefulness

**Comment by jonaskohl** [comment]
This should have already been fixed with version 3.16.0.0 and later. If this issue persists, please [open a bug report](https://github.com/jonaskohl/CapsLockIndicator/issues/new)
